docker-compose.yml

```diff
@@ -18,10 +14,6 @@ services:
     stop_signal: SIGINT
   db:
     image: mongo:latest
-    volumes:
```

Comment:
Is this a typo? You removed `volumes:` but not it's content.

Reply:
Yes. Thanks!

Reply:
Fixed